pkg/storage/unified/resource/index_server.go

```diff
@@ -86,10 +88,21 @@ func (is *IndexServer) Init(ctx context.Context, rs *server) error {
 	return nil
 }
 
-func NewResourceIndexServer() ResourceIndexServer {
-	return &IndexServer{
-		log: slog.Default().With("logger", "index-server"),
+func NewResourceIndexServer(cfg *setting.Cfg) ResourceIndexServer {
+	// register metrics
```

Comment:
seems this comment is in the wrong place?

Reply:
woops! removed 👍